File: an2_sem4/MPP/Homework/sapt11/java/network/src/main/java/ro/ubbcluj/cs/stefnmubb/festivalsellpoint/network/objectprotocol/FilterSpectacoleResponse.java
package ro.ubbcluj.cs.stefnmubb.festivalsellpoint.network.objectprotocol;

import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.domain.Spectacol;

public class FilterSpectacoleResponse extends SpectacoleResponse{
    public FilterSpectacoleResponse(){}
    public FilterSpectacoleResponse(Spectacol[] spectacole) {
        super(spectacole);
    }
}

File: an2_sem4/MPP/Homework/sapt9/cs/FestivalSellpoint/FestivalSellpoint.Network/ObjectProtocol/FilterSpectacoleResponse.cs
using FestivalSellpoint.Domain;
using System;

namespace FestivalSellpoint.Network.ObjectProtocol
{
    [Serializable]
    internal class FilterSpectacoleResponse : SpectacoleResponse
    {
        public FilterSpectacoleResponse(Spectacol[] spectacole) : base(spectacole){ }
    }
}
